### ISSUE TITLE
investigate integration-test/marathon-api-docker/build.sbt

### ISSUE THREAD

**Issue by pjfanning** [apache/incubator-pekko-management#28]
* we need to get the pekko jars from testing from MavenLocal repo instead of from official releases
* we want to test what we've just built
* see the '1.1.4' version parameter to see what needs to change

**Comment by nvollmar** [comment]
Checking the Readme, the following projects are/were not maintained by Lightbend and were left to the community:

akka-discovery-marathon-api
akka-discovery-aws-api
akka-discovery-aws-api-async
akka-discovery-consul

We might want to discuss how to go forward with them: Drop them, leave them as is or start maintaining them?

**Comment by mdedetrich** [comment]
On a similar note we should also update the test at https://github.com/apache/incubator-pekko-management#project-status (everything at Pekko is "community", this language is only relevant when it was at Lightbend).

**Comment by nvollmar** [comment]
The marathon integration test projects don't contain any testing code and look more like sample implementations.
The aws integration test project has some sbt project that is missing dependencies to even be loaded.

I'd reword the README to mark them as experimental (at best) and see if there is even interest in them => #34
